tests/test_analogue.py

```diff
@@ -95,11 +95,12 @@ def test_analogue_input(pytestconfig, board, config):
     duration = analogue_duration(pytestconfig.getoption("level"), short_test)
     fail_str = ""
 
+    samp_freqs = [f for f in features["samp_freqs"] if f <= 96000] # TODO Extend to 192KHz, 10ch as part of ADAT output for SMUX > 1 testing
```

Comment:
Shouldn't this only be if the config has ADAT? We can still test >96000 on non-ADAT configs, can't we?